stringfuzz/transformers/translate.py

```diff
@@ -4,22 +4,40 @@
 
 import random
 import string
+import copy
 
 from stringfuzz.ast import StringLitNode
 from stringfuzz.ast_walker import ASTWalker
-from stringfuzz.parser import parse
+from stringfuzz import ALL_CHARS
 
 __all__ = [
     'translate',
+    'CHARACTER_SETS',
+    'DEFAULT_CHARACTER_SET'
 ]
 
+NON_INTEGERS = 'non-integers' 
+LETTERS      = 'letters'
+ALL_SYMBOLS  = 'all'
+
+DEFAULT_CHARACTER_SET = NON_INTEGERS
+
+CHARACTER_SETS = {
+    NON_INTEGERS: [c for c in ALL_CHARS if not c.isdecimal()],
```

Comment:
So, now that I look more deeply into it, `ALPHABET` is actually not the alphabet we support. We support way more than it, because we have `[^"]` as a character class, and I wrote that constant incorrectly.

We should either restrict the scanner to a more constrained character set, or redefine `ALPHABET` to reflect what it actually accepts (which is basically every character ever). Let's discuss this tomorrow.